### PR TITLE
authentication with twitter

### PR DESCRIPTION
**Pull request by neilrioszamora**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  devise :omniauthable, :omniauth_providers => %i[facebook google_oauth2]
+  devise :omniauthable, :omniauth_providers => %i[facebook google_oauth2 twitter]
```
Comment (OtavioHenrique) [inline app/models/user.rb]: If you merge, will have a conflict, on the other pr the same line of code is:

```devise :omniauthable, :omniauth_providers => %i[facebook google_oauth2 github]```

File: app/controllers/users/omniauth_callbacks_controller.rb [hunk 2]
```diff
@@ -48,6 +48,18 @@ def google_oauth2
       redirect_to new_user_registration_url
     end
   end
+
+  def twitter
```
Comment (OtavioHenrique) [inline app/controllers/users/omniauth_callbacks_controller.rb]: Same

**Comment by OtavioHenrique** [comment]
I think that is better you merge the two branchs and open a unique pull request with all changes.

**Comment by neilrioszamora** [comment]
Here's the merged one https://github.com/J-Cunha/cryptocoinsmap/pull/14

On Wed, Sep 13, 2017 at 8:33 PM, Otavio dos Passos <notifications@github.com

**Pull request closed without merging**